recipes/google-cloud-cpp-compute/install-libgoogle-cloud.bat

```diff
@@ -0,0 +1,22 @@
+@echo on
+setlocal EnableDelayedExpansion
+
+:: CMake does not like paths with \ characters
+set LIBRARY_PREFIX="%LIBRARY_PREFIX:\=/%"
+set BUILD_PREFIX="%BUILD_PREFIX:\=/%"
+set SRC_DIR="%SRC_DIR:\=/%"
+
+:: Once DLLs are working, we should install the non-devel packages using
+::     cmake --install .b --component google_cloud_cpp_runtime
+:: and the devel packages using
+::     cmake --install .b --component google_cloud_cpp_development
+
+if [%PKG_NAME%] == [libgoogle-cloud-compute-devel] (
+  cmake --install .b
```

Comment:
here we do need to refer back to the folder defined in `bld.bat`, and it's good if it's self-explanatory.

Reply:
Changed. FWIW, I used a very short name to workaround the problems outlined in  https://github.com/googleapis/google-cloud-cpp/pull/13351